main/src/ca/uwaterloo/flix/language/ast/TypeConstructor.scala

```diff
@@ -242,6 +244,20 @@ object TypeConstructor {
     def kind: Kind = Kind.Star
   }
 
+  /**
+   * A type constructor that represents the type of a Java constructor.
+   * */
+  case class JvmConstructor(constructor: Method) extends TypeConstructor {
```

Comment:
Method -> Constructor (imported from java.lang.reflect).

Reply:
Do we consider any type parameter or does it have to specified ?

Reply:
For JvmMethod, do we then instead of a type a java method too ?

Reply:
We deal with type parameters later.